ts/screens/profile/PreferencesScreen.tsx

```diff
@@ -194,6 +195,16 @@ class PreferencesScreen extends React.Component<Props, State> {
               onPress={this.handleEmailOnPress}
             />
 
+            <ListItemComponent
+              title={I18n.t(
+                "profile.preferences.list.send_email_messages.title"
+              )}
+              subTitle={I18n.t(
+                "profile.preferences.list.send_email_messages.subtitle"
```

Comment:
Could we show a preview about the preference status?
Does it make sense?
ex:
- "abilitato per tutti i servizi"
- "disabilito per tutti i servizi"
- "personalizzata"

Reply:
updated within https://github.com/pagopa/io-app/pull/1372/commits/d88e6cab8a9525e089548d19fb86d0634773f684